src/main/java/org/jitsi/jicofo/bridge/BridgeSelectionStrategy.java

```diff
@@ -41,10 +41,9 @@ public Bridge select(
         {
             Bridge bridge
                 = selectInitial(bridges, participantRegion);
-            if (logger.isDebugEnabled())
-            {
-                logger.debug("Selected initial bridge " + bridge);
-            }
+            logger.info("Selected initial bridge " + bridge
```

Comment:
Should we log some indication of the loads (packet rates) of the other bridges?  Are we exposing that in any way?

Reply:
We can, but we expect to have 600 of them...

Reply:
Ok, so maybe we say this is nice to have but we won't be able to fully answer the question of "why did jicofo pick this bridge and not another" until we move this to the bridge-side?